src/test/java/hudson/plugins/git/BranchTest.java

```diff
@@ -1,14 +1,13 @@
 package hudson.plugins.git;
 
-import org.eclipse.jgit.lib.ObjectId;
-import org.eclipse.jgit.lib.ObjectIdRef;
-import org.eclipse.jgit.lib.Ref;
-
-import org.junit.Test;
 import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.*;
 
 import nl.jqno.equalsverifier.EqualsVerifier;
+import org.eclipse.jgit.lib.ObjectId;
```

Comment:
Thanks for the contribution!  Much appreciated!

Please don't make formatting changes like this without checking with the maintainers.  Refer to the [CONTRIBUTING file](https://github.com/jenkinsci/git-client-plugin/blob/master/CONTRIBUTING.md) where it says:

> Try to maintain reasonable consistency with the existing files.

> Please don't perform wholesale reformatting of a file without discussing with the current maintainers.

Formatting changes create a "diff wall" that make other changes (from other pull requests) more likely to have conflicts, without adding significant value to code comprehension.

When I first became a maintainer, I asked if we could switch to using automatically maintained code formatting.  I was told "no", because of the barrier it creates to contributions and the barrier it creates to reviewing history.

We may revisit that in the future, after reducing the number of open pull requests, but I'm not ready to revisit that as part of this pull request.